src/components/CopyTextToClipboard.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Text, TouchableOpacity} from 'react-native';
+import {Checkmark, Clipboard as ClipboardIcon} from './Icon/Expensicons';
+import Clipboard from '../libs/Clipboard';
+import Icon from './Icon';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /** The text to display and copy to the clipboard */
+    text: PropTypes.string.isRequired,
+
+    /** Styles to apply to the text */
+    textStyles: PropTypes.arrayOf(PropTypes.object),
+};
+
+const defaultProps = {
+    textStyles: [],
+};
+
+class CopyTextToClipboard extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.copyToClipboard = this.copyToClipboard.bind(this);
+
+        this.state = {
+            showCheckmark: false,
+        };
+    }
+
+    copyToClipboard() {
+        Clipboard.setString(this.props.text);
+        this.setState({showCheckmark: true}, () => {
+            setTimeout(() => {
+                this.setState({showCheckmark: false});
+            }, 2000);
```

Comment:
It was just an arbitrary value that I chose, I don't know that it makes much of a difference, but it would be interesting to discuss at some point if we wanted to standardize our animations/delays at some point. I do see that the default growl duration is also 2 seconds.